cdi-unit-tests/src/test/java/org/jglue/cdiunit/TestContextController.java

```diff
@@ -62,7 +62,7 @@ public Integer call() throws Exception {
 
 	@Produces
 	@RequestScoped
-	private TestCounter createTestCounter() {
+	TestCounter createTestCounter() {
```

Comment:
Looks like this fixes these tests which have been breaking in Travis:

    testAsynchronousExecution(org.jglue.cdiunit.TestContextController)
    testSynchronousExecution(org.jglue.cdiunit.TestContextController)